### PR TITLE
OVDB-71: Update INSTALL file minimum versions and add link to build documentation

### PR DESCRIPTION
**Pull request by danrbailey**
There's a good argument to remove this file entirely, but for the time being, just update the versions and add a link to point to the CMake build documentation.

### CODE REVIEW

File: openvdb/INSTALL [hunk 1]
```diff
@@ -52,9 +51,9 @@ Optional:
 
 - OpenGL 3.2 or later, for the OpenVDB viewer
 
-- Python 2.5, 2.6 or 2.7, for the Python module
+- Python 2.7, for the Python module
```
Comment (pcucka) [inline openvdb/INSTALL]: This can be "2.7 or later", since we support at least some versions of Python 3.

File: openvdb/INSTALL [hunk 2]
```diff
@@ -64,6 +63,11 @@ Other versions might work but have not been tested.
 
 Installation
 ------------
+
+This documentation is for building with GNU Make, it is recommended to use
```
Comment (pcucka) [inline openvdb/INSTALL]: Should this be worded more strongly, to say that Make support is deprecated and will be dropped in a future release?

File: openvdb/INSTALL [hunk 3]
```diff
@@ -52,9 +51,9 @@ Optional:
```
Comment (pcucka) [inline openvdb/INSTALL]: I think we just dropped support for GLFW 2.  The minimum version is now 3.0.

File: openvdb/INSTALL [hunk 4]
```diff
@@ -4,39 +4,38 @@ Installing OpenVDB
 Requirements
```
Comment (pcucka) [inline openvdb/INSTALL]: Maybe add a link here to the new Requirements web page?

File: openvdb/INSTALL (hunk 1 above)
Comment (danrbailey) [reply openvdb/INSTALL]: I thought the point in this file was to only include what we explicitly test for:

`Other versions might work but have not been tested.`

Nick fixed an issue in Python 3 recently (#428) that was reported to us - https://github.com/AcademySoftwareFoundation/openvdb/issues/427. I'm not sure we should include anything here that we don't know is actively being tested, ideally through CI?

**Comment by danrbailey** [comment]
Thanks @pcucka, just addressed your feedback.

File: openvdb/INSTALL (hunk 1 above)
Comment (pcucka) [reply openvdb/INSTALL]: Perhaps, but then we should really drop all of the "or later" qualifiers.

We've supported Python 3 for some time now (see, e.g., PR #78 and the Fedora [openvdb-python3](https://apps.fedoraproject.org/packages/openvdb-python3) package).

File: openvdb/INSTALL [hunk 5]
```diff
@@ -64,8 +69,10 @@ Other versions might work but have not been tested.
 Installation
 ------------
 
-This documentation is for building with GNU Make, it is recommended to use
-CMake to build OpenVDB. See the build documentation here -
+This documentation is for building with GNU Make which is now deprecated, it
+is strongly recommended to use CMake to build OpenVDB as this will be removed
```
Comment (pcucka) [inline openvdb/INSTALL]: How about "It is strongly recommended to use CMake to build OpenVDB as the Makefiles will be removed in an upcoming release"?

**Comment by danrbailey** [comment]
@pcucka - just addressed your last two comments.

**Pull request merged**